core/domain/draft_upgrade_services.py

```diff
@@ -306,6 +306,27 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v55_dict_to_v56_dict(
+        cls, draft_change_list: List[exp_domain.ExplorationChange]
+    ) -> List[exp_domain.ExplorationChange]:
+        """Converts draft change list from state version 55 to 56. Version 56
+        adds an inapplicable_skill_misconception_ids list property to the
+        state.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+
+        Raises:
+            InvalidDraftConversionException. The conversion cannot be
+                completed.
+        """
+        return draft_change_list
```

Comment:
I'm not really seeing where this exception could be raised. It seems that this is just an identity function. Is there some behavior that's supposed to go here?

Reply:
It's meant to be an identity function, this migration shouldn't make any changes to the current draft change lists. The docstring is there following the pattern of previous draft conversion functions, some of which were also identity functions. Hope that makes sense.